translator/translate/logs/util/filter_retention.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"fmt"
+	"github.com/aws/amazon-cloudwatch-agent/translator"
+	"strings"
+)
+
+func CheckForConflictingRetentionSettings(logConfigs []interface{}, currPath string) []interface{} {
```

Comment:
Also a nitpick, but this name is very verbose. While on the one hand, I'd rather be verbose than ambiguous, I think that the name is really long and we could make the function name more succinct.

```suggestion
func ValidateLogRetentionSettings(logConfigs []interface{}, currPath string) []interface{} {
```